apps/api_web/lib/api_web/controllers/alert_controller.ex

```diff
@@ -315,14 +315,34 @@ defmodule ApiWeb.AlertController do
               example: ["BOARD", "EXIT"]
             )
 
-            facility(:string, "Unique id of a facility", example: "405")
-            route(:string, "Unique id of a route", example: "CR_Worcester")
-            route_type(:integer, route_type_description(), example: 2)
-            stop(:string, "Unique id of a stop", example: "Auburndale")
-            trip(:string, "Unique id of a trip", example: "CR-Weekday-Spring-17-517")
-          end
+            direction_id(
+              nullable(direction_id_schema(), true),
+              "`direction_id` of the affected Trip.\n\n" <> direction_id_description(),
+              example: 0
+            )
+
+            facility(nullable(%Schema{type: :string}, true), "`id` of the affected Facility.",
+              example: "405"
+            )
+
+            route(nullable(%Schema{type: :string}, true), "`id` of the affected Route.",
+              example: "CR_Worcester"
```

Comment:
Extremely minor nitpick about something carried over from the old version, but if we want the example to be an actual route it should be `CR-Worcester` with a hyphen rather than an underscore.

Reply:
Good catch! I fixed it 😊